artist/util/config_dictionary.py

```diff
@@ -282,7 +282,10 @@
 paint_first_axis = "axis_1_motor_position"
 """Key to access the first motor position in a ``PAINT`` calibration.json file."""
 paint_second_axis = "axis_2_motor_position"
```

Comment:
```suggestion
paint_second_axis = "axis_2_motor_position"
"""Key to access the second motor position in a ``PAINT`` calibration.json file."""
```

(This was missed in an earlier PR. Let's include it now.)